packages/e2e-tests/next-app/cypress/integration/redirects.test.ts

```diff
@@ -42,7 +42,7 @@ describe("Redirects Tests", () => {
       {
         path: "//example.com/",
         expectedPath: "/example.com",
-        expectedStatus: 404
+        expectedStatus: 308
```

Comment:
This was not passing after the update to cypress - the 308 is returned initially and the 404 is the status code of the subsequent request.